styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3c3a39;
+  height: 60px;
+}
+
+.nav_logo {
+  text-decoration: none;
+  color: white;
+  padding: 10px;
+  padding-top: 20px;
+  font-size: 25px;
+}
+
+.nav_logo1 {
+  margin-top: 15px;
+}
+
+.navbar1 {
+  margin: 15px;
+}
+
+.nav_toggle {
+  color: white;
+  font-size: 30px;
+}
+
+.nav_links {
+  transition: 600ms;
+  font-weight: bold;
+}
+
+.nav_links:hover {
+  background-color: #a5a58d;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 100vh;
+  background-color: black;
+  color: #ff6b00;
+  padding: 110px 10px 0 20px;
+  gap: 0.5rem;
+  background-image: url("background.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
kindly consider using the same color as was requested by the previous reviewer, (padding, margin, font size), and all the rest as per the original design, take a look at this [Figma template](https://user-images.githubusercontent.com/79058364/159028568-4c195e98-3576-45f3-8176-667cf86919dc.png)